router/forwarder.go

```diff
@@ -38,11 +38,11 @@ func (conn *LocalConnection) ensureForwarders() error {
 	}
 
 	forwarder := NewForwarder(conn, encryptor, udpSender, DefaultPMTU)
-	forwarderDF := NewForwarder(conn, encryptorDF, udpSenderDF, DefaultPMTU)
+	forwarderDF := NewForwarderDF(conn, encryptorDF, udpSenderDF, DefaultPMTU)
 	effectivePMTU := forwarderDF.unverifiedPMTU
 	//NB: only forwarderDF can ever encounter EMSGSIZE errors, and
 	//thus perform PMTU verification
```

Comment:
I think that comment is now somewhat out of place. It previously served as an explanation of why we called `Start` with two different arguments.